data-prepper-expression/build.gradle

```diff
@@ -38,7 +38,7 @@ jacocoTestCoverageVerification {
     violationRules {
         rule { //in addition to core projects rule - this one checks for 100% code coverage for this project
             limit {
-                minimum = 1.0 //keep this at 100%
+                minimum = 0.98 //keep this at 100%
```

Comment:
Are you following up to bring this back to 100 or is this latest change permanent? If it's permanent, let remove this comment.

Reply:
Added back in [this PR](https://github.com/opensearch-project/data-prepper/pull/1166/files#diff-0dddda0580c72f23a84a7b85ed7a151fc951a45ae09cd7d2861566213a3fe7f4R41)